src/libraries/CurrencyDelta.sol

```diff
@@ -6,34 +6,33 @@ import {Currency} from "../types/Currency.sol";
 library CurrencyDelta {
     uint256 constant CURRENCY_DELTA_SLOT = uint256(keccak256("CurrencyDelta")) - 1;
 
-    function setCurrencyDelta(address locker, Currency currency, int256 delta) internal {
+    function _computeKey(address locker, Currency currency) internal pure returns (bytes32)
+    {
         uint256 slot = CURRENCY_DELTA_SLOT;
-
+        bytes32 hashSlot;
```

Comment:
i think it would be neater to define this in the function signature as the return value :)

Reply:
like `returns (bytes32 hashSlot)`

Reply:
and then lets align naming between the function name and the return value

so either `computeSlot` and `bytes32 slot`
or `computeKey` and `bytes32 key`

I would lean towards the former